src/bentoml/cloud.py

```diff
@@ -1,3 +1,12 @@
 from ._internal.cloud import BentoCloudClient as BentoCloudClient
-from ._internal.cloud import Resource as Resource
 from ._internal.cloud import YataiClient as YataiClient
+
+deprecated_names = ["Resource"]
+
+
+def __getattr__(name: str):
+    if name in deprecated_names:
```

Comment:
This defeats the whole purpose of deprecation right? this is still breaking.

Reply:
Oh, I misunderstood. Just realized you said throwing warning not exception, but I believe we are not going to provide backward compatibility for this either, just like CLI?